src/seedu/addressbook/data/person/Address.java

```diff
@@ -11,6 +11,10 @@
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public Block addressBlock;
```

Comment:
Why public? Should these attributes be accesses directly or via getters? Think about OOP principle, especially encapsulation.